2019-10-25 23:22:09,339 - [service.py: 36] -INFO: waiting for connections...
2019-10-25 23:24:59,323 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:7373...
2019-10-25 23:25:00,328 - [service.py: 25] -INFO: b'4444'
2019-10-26 00:01:01,562 - [service.py: 36] -INFO: waiting for connections...
2019-10-26 00:02:26,446 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:9977...
2019-10-26 00:02:27,450 - [service.py: 25] -INFO: b'4444'
2019-10-26 00:02:28,528 - [service.py: 29] -INFO: Connection from 127.0.0.1:9977 closed.
2019-10-26 00:03:25,452 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:10029...
2019-10-26 00:03:26,459 - [service.py: 25] -INFO: b'4444'
2019-10-26 00:16:52,070 - [service.py: 29] -INFO: Connection from 127.0.0.1:10029 closed.
2019-10-26 00:17:39,291 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:10908...
2019-10-26 00:17:40,294 - [service.py: 25] -INFO: b'4444'
2019-10-26 00:24:10,092 - [service.py: 36] -INFO: waiting for connections...
2019-10-26 00:24:32,739 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:11344...
2019-10-26 00:24:33,747 - [service.py: 25] -INFO: b'4444'
2019-10-26 14:03:49,683 - [service.py: 36] -INFO: waiting for connections...
2019-10-26 14:04:46,168 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:6075...
2019-10-26 14:04:47,170 - [service.py: 25] -INFO: b'4444'
2019-10-26 23:50:18,076 - [service.py: 29] -INFO: Connection from 127.0.0.1:6075 closed.
2019-10-26 23:51:58,219 - [service.py: 36] -INFO: waiting for connections...
2019-10-26 23:52:38,137 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:15731...
2019-10-26 23:52:39,139 - [service.py: 25] -INFO: b'4444'
2019-10-28 10:57:22,684 - [service.py: 36] -INFO: waiting for connections...
2019-10-28 10:58:06,917 - [service.py: 6] -INFO: Accept new connection from 127.0.0.1:5822...
2019-10-28 10:58:07,920 - [service.py: 25] -INFO: b'4444'
